packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/column-splitter/column-splitter.component.html

```diff
@@ -0,0 +1,13 @@
+﻿<div [ngClass]="['row', 'component', 'column-splitter', columnSplitterStyles]" [attr.id]="id || null">
```

Comment:
"class" attribute is not computed, so it doesn't make sense to use "[ngClass]"
Do we need to set a fallback value for "id" to "null"? Do we even need to set a fallback value?
```suggestion
﻿<div class="row component column-splitter columnSplitterStyles" [attr.id]="id || null">
```